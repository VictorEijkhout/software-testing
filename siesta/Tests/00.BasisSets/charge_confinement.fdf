%include ../water_box.fdf
SystemLabel         charge_confinement
NumberOfAtoms       3
NumberOfSpecies     2

%block ChemicalSpeciesLabel
 1  8  O
 2  1  H
%endblock ChemicalSpeciesLabel

%block PAO.Basis
O    3
n=2   0   3   E   50. 8.0
    8.0   3.5   2.0
n=2   1   3   Q   0.0 0.0 0.1  E   10. 8.0
    8.0   3.5   2.0
n=3   2   1   E   50. 9.0 Q  8.5 0.4
    8.0
H    2
n=1   0   3   E   50. 8.0
    8.0   3.5   2.0
n=2   1   1   E   50. 9.0 Q  5.5 0.5
    8.0
%endblock PAO.Basis
AtomicCoordinatesFormat  Ang
%block AtomicCoordinatesAndAtomicSpecies
 0.000  0.000  0.000  1
 0.757  0.586  0.000  2
-0.757  0.586  0.000  2
%endblock AtomicCoordinatesAndAtomicSpecies
