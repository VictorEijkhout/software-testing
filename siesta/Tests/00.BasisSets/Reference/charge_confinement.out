Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/00.BasisSets/work/charge_confinement
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:35:05

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.98818
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel         charge_confinement
NumberOfAtoms       3
NumberOfSpecies     2
%block ChemicalSpeciesLabel
 1  8  O
 2  1  H
%endblock ChemicalSpeciesLabel
%block PAO.Basis
O    3
n=2   0   3   E   50. 8.0
    8.0   3.5   2.0
n=2   1   3   Q   0.0 0.0 0.1  E   10. 8.0
    8.0   3.5   2.0
n=3   2   1   E   50. 9.0 Q  8.5 0.4
    8.0
H    2
n=1   0   3   E   50. 8.0
    8.0   3.5   2.0
n=2   1   1   E   50. 9.0 Q  5.5 0.5
    8.0
%endblock PAO.Basis
AtomicCoordinatesFormat  Ang
%block AtomicCoordinatesAndAtomicSpecies
 0.000  0.000  0.000  1
 0.757  0.586  0.000  2
-0.757  0.586  0.000  2
%endblock AtomicCoordinatesAndAtomicSpecies
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: Water molecule
reinit: -----------------------------------------------------------------------
reinit: System Label: charge_confinement
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:    8 Label: O
Species number:   2 Atomic number:    1 Label: H


---- Processing specs for species: O
Ground state valence configuration:   2s02  2p04
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/O.psf

---- Processing specs for species: H
Ground state valence configuration:   1s01
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/H.psf

---- Pseudopotential check for O 

Pseudized shells:
2s( 2.00) rc: 1.14
2p( 4.00) rc: 1.14
3d( 0.00) rc: 1.14
4f( 0.00) rc: 1.14
Valence configuration for ps generation: (assumed as above)

---- Pseudopotential check for H 

Pseudized shells:
1s( 1.00) rc: 1.25
2p( 0.00) rc: 1.25
3d( 0.00) rc: 1.25
4f( 0.00) rc: 1.25
Valence configuration for ps generation: (assumed as above)
For O, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
For H, standard SIESTA heuristics set lmxkb to 2
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
O                    Z=   8    Mass=  15.999        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=2
          i=1  nzeta=3  polorb=0  (2s)
            splnorm:   0.15000    
               vcte:    50.000    
               rinn:    8.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    8.0000      3.5000      2.0000    
            lambdas:    1.0000      1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=3  polorb=0  (2p)
            splnorm:   0.15000    
               vcte:    10.000    
               rinn:    8.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000    
                rcs:    8.0000      3.5000      2.0000    
            lambdas:    1.0000      1.0000      1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3d)
            splnorm:   0.15000    
               vcte:    50.000    
               rinn:    9.0000    
               qcoe:    8.5000    
               qyuk:   0.40000    
               qwid:   0.10000E-01
                rcs:    8.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for O                     (Z =   8)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    6.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.1366
V l=1 = -2*Zval/r beyond r=  1.1332
V l=2 = -2*Zval/r beyond r=  1.1298
V l=3 = -2*Zval/r beyond r=  1.1264
All V_l potentials equal beyond r=  1.1355
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.1366
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     33.667 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     76.727 Ry
atom: Maximum radius for chlocal:    1.37241
atom: Maximum radius for r*vlocal+2*Zval:    1.25469

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.293209   el= -1.742414   Ekb=  9.076872   kbcos=  0.325320
GHOST: No ghost state for L =  1
   l= 1   rc=  1.284119   el= -0.676589   Ekb= -8.194485   kbcos= -0.396010
GHOST: No ghost state for L =  2
   l= 2   rc=  1.433152   el=  0.013307   Ekb= -2.071665   kbcos= -0.003690
GHOST: No ghost state for L =  3
   l= 3   rc=  1.540891   el=  0.019584   Ekb= -0.817137   kbcos= -0.000371

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 2s
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    8.001114
                 energy =   -1.742414
                kinetic =    1.522903
    potential(screened) =   -3.265317
       potential(ionic) =  -11.137958

   izeta = 2
                 rmatch =    3.504263
              splitnorm =    0.042440
                 energy =   -1.703033
* WARNING: effective split_norm is quite small. Orbitals will be very similar.
                kinetic =    1.814097
    potential(screened) =   -3.517130
       potential(ionic) =  -11.672986

   izeta = 3
                 rmatch =    2.001398
              splitnorm =    0.266324
                 energy =   -0.918652
                kinetic =    3.372859
    potential(screened) =   -4.291511
       potential(ionic) =  -13.197670

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    8.001114
                 energy =   -0.676568
                kinetic =    4.738883
    potential(screened) =   -5.415452
       potential(ionic) =  -13.028204

   izeta = 2
                 rmatch =    3.504263
              splitnorm =    0.113934
                 energy =   -0.608473
                kinetic =    5.846342
    potential(screened) =   -6.454815
       potential(ionic) =  -14.635002

   izeta = 3
                 rmatch =    2.001398
              splitnorm =    0.375301
                 energy =    0.131352
                kinetic =    9.722310
    potential(screened) =   -9.590958
       potential(ionic) =  -18.747172

SPLIT: Orbitals with angular momentum L= 2

SPLIT: Basis orbitals for state 3d
Charge confinement: Q:     8.5000
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    8.001114
                 energy =    2.943919
                kinetic =    5.678529
    potential(screened) =   -2.734610
       potential(ionic) =   -9.457296
atom: Total number of Sankey-type orbitals: 17

atm_pop: Valence configuration (for local Pseudopot. screening):
 2s( 2.00)                                                            
 2p( 4.00)                                                            
 3d( 0.00)                                                            
Vna: chval, zval:    6.00000   6.00000

Vna:  Cut-off radius for the neutral-atom potential:   8.001114

atom: _________________________________________________________________________

<basis_specs>
===============================================================================
H                    Z=   1    Mass=  1.0080        Charge= 0.17977+309
Lmxo=1 Lmxkb= 2    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=1
          i=1  nzeta=3  polorb=0  (1s)
            splnorm:   0.15000    
               vcte:    50.000    
               rinn:    8.0000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    8.0000      3.5000      2.0000    
            lambdas:    1.0000      1.0000      1.0000    
L=1  Nsemic=0  Cnfigmx=2
          i=1  nzeta=1  polorb=0  (2p)
            splnorm:   0.15000    
               vcte:    50.000    
               rinn:    9.0000    
               qcoe:    5.5000    
               qyuk:   0.50000    
               qwid:   0.10000E-01
                rcs:    8.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for H                     (Z =   1)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    1.00000

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  1.2384
V l=1 = -2*Zval/r beyond r=  1.2321
V l=2 = -2*Zval/r beyond r=  1.2272
All V_l potentials equal beyond r=  1.2371
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  1.2384
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside     28.363 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     64.639 Ry
atom: Maximum radius for chlocal:    1.44620
atom: Maximum radius for r*vlocal+2*Zval:    1.31026

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  1.364142   el= -0.467325   Ekb= -2.009475   kbcos= -0.336680
GHOST: No ghost state for L =  1
   l= 1   rc=  1.430269   el=  0.007562   Ekb= -0.503817   kbcos= -0.022554
GHOST: No ghost state for L =  2
   l= 2   rc=  1.469686   el=  0.013286   Ekb= -0.191752   kbcos= -0.002430

KBgen: Total number of Kleinman-Bylander projectors:   9
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:
atom: Selected multiple-zeta basis: split     

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 1s
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    8.001114
                 energy =   -0.467037
                kinetic =    0.826832
    potential(screened) =   -1.293869
       potential(ionic) =   -1.818468

   izeta = 2
                 rmatch =    3.504263
              splitnorm =    0.233106
                 energy =   -0.273272
                kinetic =    1.697835
    potential(screened) =   -1.971106
       potential(ionic) =   -2.550292

   izeta = 3
                 rmatch =    2.001398
              splitnorm =    0.549333
                 energy =    0.847274
                kinetic =    3.914177
    potential(screened) =   -3.066903
       potential(ionic) =   -3.676605

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 2p
Charge confinement: Q:     5.5000
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    8.001114
                 energy =    1.532681
                kinetic =    2.592843
    potential(screened) =   -1.060162
       potential(ionic) =   -1.592121
atom: Total number of Sankey-type orbitals:  6

atm_pop: Valence configuration (for local Pseudopot. screening):
 1s( 1.00)                                                            
 2p( 0.00)                                                            
Vna: chval, zval:    1.00000   1.00000

Vna:  Cut-off radius for the neutral-atom potential:   8.001114

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1    8 O                       # Species index, atomic number, species label
    2    1 H                       # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
O                     3                    # Species label, number of l-shells
 n=2   0   3                         # n, l, Nzeta 
   8.001      3.504      2.001   
   1.000      1.000      1.000   
 n=2   1   3                         # n, l, Nzeta 
   8.001      3.504      2.001   
   1.000      1.000      1.000   
 n=3   2   1                         # n, l, Nzeta 
   8.001   
   1.000   
H                     2                    # Species label, number of l-shells
 n=1   0   3                         # n, l, Nzeta 
   8.001      3.504      2.001   
   1.000      1.000      1.000   
 n=2   1   1                         # n, l, Nzeta 
   8.001   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file O.ion.nc
Dumping basis to NetCDF file H.ion.nc
coor:   Atomic-coordinates input format  =     Cartesian coordinates
coor:                                          (in Angstroms)

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      1.43052   1.10738   0.00000  2        2
siesta:     -1.43052   1.10738   0.00000  2        3

siesta: System type = chain     

initatomlists: Number of atoms, orbitals, and projectors:      3    29    34

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        2
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   100.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =        3
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   299.9863 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 2
mix.SCF:    Linear mixing weight                 =     0.250000
mix.SCF:    Mixing weight                        =     0.250000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.2500
  weight.linear 0.2500
  history 2
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000

siesta: k-grid: Number of k-points = 1
siesta: k-point is Gamma-only

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

outcell: Unit cell vectors (Ang):
       10.000000    0.000000    0.000000
        0.000000   10.000000    0.000000
        0.000000    0.000000    8.000000

outcell: Cell vector modules (Ang)   :   10.000000   10.000000    8.000000
outcell: Cell angles (23,13,12) (deg):     90.0000     90.0000     90.0000
outcell: Cell volume (Ang**3)        :    800.0000
Gamma-point calculation with interaction between periodic images
Some features might not work optimally:
e.g. DM initialization from atomic data
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=29 nrows=29 sparsity=1.0000 nnzs=841, refcount: 7>
  <dData1D:(new from dSpData1D) n=841, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=29 nrows=29 sparsity=1.0000 nnzs=841, refcount: 8>
  <dData2D:DM n=841 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      29

InitMesh: MESH = 64 x 64 x 50 = 204800
InitMesh: Mesh cutoff (required, used) =   100.000   107.960 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -61.051282
siesta: Eions   =       810.513381
siesta: Ena     =       182.469803
siesta: Ekin    =       401.723742
siesta: Enl     =       -77.976430
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -45.520946
siesta: DUscf   =         6.787609
siesta: DUext   =         0.000000
siesta: Ex      =      -107.878327
siesta: Ec      =       -12.937793
siesta: Exc     =      -120.816120
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -459.372796
siesta: Etot    =      -463.845724
siesta: FreeEng =      -463.845724

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -459.372796     -463.845724     -463.845724  0.974852 -9.152123 15.883649
timer: Routine,Calls,Time,% = IterSCF        1       0.028   1.49
   scf:    2     -468.609354     -466.669989     -466.669989  0.665518 -6.490983  5.884665
   scf:    3     -467.099140     -467.048920     -467.048920  0.554004 -4.758294  0.690401
SCF_NOT_CONV: SCF did not converge  in maximum number of steps.
Geom step, scf iteration, dmax:    0     3     0.050744

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:       3      29

siesta: E_KS(eV) =             -467.0489

siesta: E_KS - E_eggbox =      -467.0489

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000   -0.048517    0.000000
----------------------------------------
   Max    0.970756
   Res    0.509994    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.970756    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):       -2.15       -1.18       -0.32        0.00       -0.00        0.00
(Free)E + p*V (eV/cell)     -466.4409
Target enthalpy (eV/cell)     -467.0489

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -111.978884
siesta: Eions   =       810.513381
siesta: Ena     =       182.469803
siesta: Ekin    =       347.685044
siesta: Enl     =       -64.135794
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =       -12.085705
siesta: DUscf   =         1.501456
siesta: DUext   =         0.000000
siesta: Ex      =       -99.441798
siesta: Ec      =       -12.528545
siesta: Exc     =      -111.970342
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -467.099140
siesta: Etot    =      -467.048920
siesta: FreeEng =      -467.048920

siesta: Final energy (eV):
siesta:  Band Struct. =    -111.978884
siesta:       Kinetic =     347.685044
siesta:       Hartree =     443.338666
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =     -99.441798
siesta:         Corr. =     -12.528545
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -111.970342
siesta:  Ion-electron =   -1199.639198
siesta:       Ion-ion =      53.536910
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -467.048920
siesta:         Fermi =      -4.758294

siesta: Atomic forces (eV/Ang):
siesta:      1   -0.000000   -0.970756    0.000000
siesta:      2    0.697572    0.461120   -0.000000
siesta:      3   -0.697572    0.461120   -0.000000
siesta: ----------------------------------------
siesta:    Tot   -0.000000   -0.048517    0.000000

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.001344    0.000000    0.000000
siesta:     0.000000   -0.000734    0.000000
siesta:    -0.000000    0.000000   -0.000202

siesta: Cell volume =        800.000000 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00000828          0.00000095  Ry/Bohr**3
siesta:           0.00076008          0.00008695  eV/Ang**3
siesta:           1.21778336          0.13931119  kBar
(Free)E+ p_basis*V_orbitals  =        -464.112518
(Free)Eharris+ p_basis*V_orbitals  =        -464.162738

siesta: Electric dipole (a.u.)  =    0.000000    0.769958    0.000000
siesta: Electric dipole (Debye) =    0.000000    1.957038    0.000000

cite: Please see "charge_confinement.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       2.066
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       2.066       2.066   100.00
Setup                  1       0.077       0.077     3.71
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       1.989       1.989    96.27
geom_init              1       1.802       1.802    87.21
state_init             1       0.233       0.233    11.29
hsparse                1       0.000       0.000     0.00
overlap                1       0.233       0.233    11.27
Setup_H0               1       1.540       1.540    74.54
naefs                  2       0.000       0.000     0.01
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.118       0.235    11.39
nlefsm                 2       0.617       1.233    59.70
DHSCF_Init             1       0.072       0.072     3.48
DHSCF1                 1       0.008       0.008     0.37
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.064       0.064     3.10
REMESH                 1       0.004       0.004     0.19
REORD                 18       0.000       0.003     0.13
PHION                  1       0.046       0.046     2.22
COMM_BSC              23       0.000       0.003     0.13
POISON                 6       0.006       0.037     1.79
fft                   12       0.002       0.030     1.45
setup_H                4       0.029       0.114     5.54
DHSCF                  5       0.043       0.213    10.33
DHSCF3                 5       0.029       0.143     6.93
rhoofd                 5       0.007       0.035     1.70
XC                     5       0.008       0.042     2.02
GXC-CellXC             5       0.008       0.040     1.94
gridxc@cellXC          5       0.008       0.040     1.93
vmat                   5       0.005       0.026     1.27
IterSCF                3       0.029       0.087     4.20
compute_dm             3       0.000       0.000     0.02
diagon                 3       0.000       0.000     0.02
r-eigvec               3       0.000       0.000     0.02
r-buildHS              3       0.000       0.000     0.00
rdiag                  3       0.000       0.000     0.02
rdiag1                 3       0.000       0.000     0.00
rdiag2                 3       0.000       0.000     0.00
rdiag3                 3       0.000       0.000     0.01
rdiag4                 3       0.000       0.000     0.00
r-buildD               3       0.000       0.000     0.00
MIXER                  3       0.000       0.000     0.00
PostSCF                1       0.100       0.100     4.84
DHSCF4                 1       0.070       0.070     3.40
dfscf                  1       0.062       0.062     2.98
overfsm                1       0.000       0.000     0.01
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.000       0.000     0.01
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:35:07
Job completed
